src/olympia/devhub/tests/test_forms.py

```diff
@@ -392,15 +393,17 @@ def test_fenix_range_disabled_range_for_regular_extensions(self):
         doc = pq(content)
         assert doc('#id_form-1-application')[0].attrib['value'] == str(amo.ANDROID.id)
         # Versions inside the forbidden Fenix range are disabled for regular
-        assert len(doc('#id_form-1-min option')) == 12
-        assert len(doc('#id_form-1-max option')) == 15
+        assert len(doc('#id_form-1-min option')) == 13
+        assert len(doc('#id_form-1-max option')) == 16
         # extensions.
```

Comment:
heh, weird comment layout

Reply:
Eh good catch, fixed in https://github.com/mozilla/addons-server/pull/21459/commits/5b05e018a0c88ca52cf0f5ec5bbdb2a095e1883a